.github/workflows/ci.yml

```diff
@@ -10,19 +10,48 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: apt-get
-      run: sudo dpkg --add-architecture i386 && sudo apt-get update && sudo apt-get install build-essential libstdc++-10-dev cmake clang libssl-dev zlib1g-dev libxxhash-dev libtbb-dev git bsdmainutils dwarfdump libc6-dev:i386 lib32gcc-10-dev libstdc++-10-dev-arm64-cross gcc-10-aarch64-linux-gnu g++-10-aarch64-linux-gnu
+      run: |
+        sudo dpkg --add-architecture i386 && sudo apt-get update && \
+        sudo apt-get install build-essential libstdc++-10-dev cmake clang \
+          libssl-dev zlib1g-dev libxxhash-dev libtbb-dev git bsdmainutils \
+          dwarfdump libc6-dev:i386 lib32gcc-10-dev libstdc++-10-dev-arm64-cross \
+          gcc-10-aarch64-linux-gnu g++-10-aarch64-linux-gnu && \
+        sudo update-alternatives --install /usr/local/bin/aarch64-linux-gnu-gcc \
+          aarch64-linux-gnu-gcc /usr/bin/aarch64-linux-gnu-gcc-10 20 && \
+        sudo update-alternatives --install /usr/local/bin/aarch64-linux-gnu-g++ \
+          aarch64-linux-gnu-g++ /usr/bin/aarch64-linux-gnu-g++-10 20
+    - name: ccache
```

Comment:
I don't want to use a third party tool here unless it is necessary, as I prefer keeping things as simple as possible. For example, that's why I stick with the good old make instead of a sophisticated build system.

In this case, ccache is used for making build faster on a buildbot. How much faster is this with this change? I don't care about bots performance too much.

Reply:
Looking at https://github.com/rui314/mold/actions/runs/999535407, roughly 2 min 58 s vs 2 s (clang build was with hot ccache and and gcc without). 3 minutes gain is quite significant IMO. It's not too much but it adds up and speeds up iterations when doing multiple force pushes into PR branch with small fixes.
If you are concerned about using third-party Github Action, then I can implement it using bare GitHub's `cache action. Though I think it would be just a code duplication and wheel reinvention.

Reply:
Hm, 3 min vs 2s. I didn't expect that the difference is so huge. I have to agree with you that this is quite significant. I'm convinced that we should use ccache.